use-cases/vehicleiot-uc/full-demo/service.yaml

```diff
@@ -114,6 +114,9 @@ topology_template:
         security_groups:  { get_input: security-groups } 
         flavor:           { get_input: flavor-name }
         username: centos
+        meta:
+          prometheus_io_scrape: true
+          prometheus_io_port: 9100
```

Comment:
Not sure this is needed, as #8 already added that metadata to sodalite.nodes.OpenStack.VM.

Reply:
Yes - I would agree with @rosogon - only changes to the input file are needed since we added security group and changed the selected image for VM creation - meta data should be already applied on VM creation.